.github/workflows/github.yml

```diff
@@ -1,17 +1,68 @@
 name: VCMI
 
 on:
-    push:
-      branches:
-        - features/*
-        - develop
-    pull_request:
+  push:
+    branches:
+      - features/*
+  pull_request:
+  schedule:
+    - cron: '0 2 * * *'
+  workflow_dispatch:
```

Comment:
Why do we need dispatch here?

Reply:
> also possible to trigger build manually now

Reply:
I understand. It is kind of generic question. Who is going to start it manually and why. Ok, let it be.

Reply:
in case a build is needed asap without waiting till night